quickstart.mdx

```diff
@@ -31,6 +31,18 @@ If you prefer to work locally, follow these steps to get started.
         <img src="/assets/console/api-keys-light.png" alt="Tilebox Console" className="dark:hidden"  />
         <img src="/assets/console/api-keys-dark.png" alt="Tilebox Console" className="hidden dark:block" />
       </Frame>
+
+      Copy the API key and keep it somewhere safe. You will need it to authenticate your requests.
+    </Step>
+    <Step title="Create a Cluster">
+      Create cluster by logging into the [Tilebox Console](https://console.tilebox.com), navigating to [Workflows -> Clusters](https://console.tilebox.com/workflows/clusters), and clicking the "Create cluster" button.
+
+      <Frame>
+        <img src="/assets/console/cluster-light.png" alt="Tilebox Console" className="dark:hidden"  />
+        <img src="/assets/console/cluster-dark.png" alt="Tilebox Console" className="hidden dark:block" />
+      </Frame>
+
+      Copy the cluster slug, you will need it to run your workflows.
```

Comment:
We could potentially highlight this as tip:

<Tip>
Copy the cluster slug, you will need it to run your workflows.
</Tip>

but I'm not sure how it would look like exactly